website/src/content/blog/biome-v1-7.md

```diff
@@ -0,0 +1,303 @@
+## Migrate from Prettier with a single command
+
+[Biome v1.6 introduced the subcommand `biome migrate prettier`](/blog/biome-v1-6/#easier-migration-from-prettier).
+
+In Biome v1.7, we add support of [Prettier's `overrides`](https://prettier.io/docs/en/configuration.html#configuration-overrides) and attempts to convert `.prettierignore` glob patterns to globs supported by Biome.
+
+During the migration, Prettier's `overrides` is translated to [Biome's `overrides`](https://biomejs.dev/reference/configuration/#overrides).
+Given the following `.prettierrc.json`
+
+```json title=".prettierrc.json"
+{
+	"useTabs": false,
+	"singleQuote": true,
+	"overrides": [
+		{
+      		"files": ["*.json"],
+      		"options": { "tabWidth": 2 }
+    	}
+	]
+}
+```
+
+Run `biome migrate prettier --write` to migrate your Prettier configuration to Biome.
+This results in the following Biome configuration:
+
+```json title="biome.json"
+{
+	"formatter": {
+		"enabled": true,
+		"formatWithErrors": false,
+		"indentStyle": "space",
+		"indentWidth": 2,
+		"lineEnding": "lf",
+		"lineWidth": 80,
+		"attributePosition": "auto"
+	},
+	"organizeImports": { "enabled": true },
+	"linter": { "enabled": true, "rules": { "recommended": true } },
+	"javascript": {
+		"formatter": {
+			"jsxQuoteStyle": "double",
+			"quoteProperties": "asNeeded",
+			"trailingComma": "all",
+			"semicolons": "asNeeded",
+			"arrowParentheses": "always",
+			"bracketSpacing": true,
+			"bracketSameLine": false,
+			"quoteStyle": "single",
+			"attributePosition": "auto"
+		}
+	},
+	"overrides": [
+		{
+			"include": ["*.json"],
+			"formatter": {
+				"indentWidth": 2
+			}
+		}
+	]
+}
+```
+
+The subcommand needs Node.js to load JavaScript configurations such as `.prettierrc.js`.
+`biome migrate prettier` doesn't support configuration written in JSON5, TOML, or YAML.
+
+
+## Emit machine-readable reports
+
+Biome is now able to output JSON reports detailing the diagnostics emitted by a command.
+
+For instance, you can emit a report when you lint a codebase:
+
+```shell
+biome lint --reporter=json-pretty .
+```
+
+For now, we support two report formats: `json` and `json-pretty`.
+
+Note that the report format is **experimental** and it might change in the future.
+Please try this feature and let us know if any information is missing from the reports.
+
+
+## Check `git` staged files
+
+Biome v1.5 added the `--changed` to format and lint `git` tracked files that have been changed.
+
+Today we are introducing a new option `--staged` which allows you to check only files that have been added to the _Git index_ (_staged files_).
+This is useful for checking that the files you want to commit are formatted and linted:
+
+```shell
+biome check --staged .
+```
+
+This is handy for writing your own [pre-commit script](/recipes/git-hooks/#shell-script).
+Note that, unstaged changes on a staged file are **not** ignored.
+Thus, we still recommend using a [dedicated pre-commit tool](/recipes/git-hooks/)..
```

Comment:
```suggestion
Biome is now able to output JSON reports detailing the diagnostics emitted by a command.

For instance, you can emit a report when you lint a codebase:

```shell
biome lint --reporter=json-pretty .
```

For now, we support two report formats: `json` and `json-pretty`.

Note that the report format is **experimental**, and it might change in the future.
Please try this feature and let us know if any information needs to be added to the reports.


## Check `git` staged files

Biome v1.5 added the `--changed` to format and lint `git` tracked files that have been changed.

Today, we are introducing a new option, `--staged`, which allows you to check only files added to the _Git index_ (_staged files_).
This is useful for checking that the files you want to commit are formatted and linted:

```shell
biome check --staged .
```

This is handy for writing your own [pre-commit script](/recipes/git-hooks/#shell-script).
Note that unstaged changes on a staged file are **not** ignored.
Thus, we still recommend using a [dedicated pre-commit tool](/recipes/git-hooks/).
```